src/seq_join.rs

```diff
@@ -177,10 +220,11 @@ where
 mod test {
     use crate::seq_join::{seq_join, seq_try_join_all};
     use futures::{
-        future::{lazy, pending, BoxFuture},
+        future::{lazy, BoxFuture},
         Future, StreamExt,
     };
-    use std::{convert::Infallible, iter::once, num::NonZeroUsize};
+    use futures_util::stream::poll_immediate;
```

Comment:
This is in `futures::stream` too.  Is there something we needed `futures_util` for?  It seems like your IDE keeps picking it up over the canonical version.

Reply:
yea I guess it is because of order it appears in the auto-import window :(